interface/patient_file/history/history_full.php

```diff
@@ -48,21 +48,11 @@
 ?>
 <html>
 <head>
-<?php html_header_show();?>
```

Comment:
I wouldn't remove the above, since it seems that some folks use this as a hook for custom stuff.